benchmarks/curl_curl_fuzzer_http/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder@sha256:dc0a0e4327c53b000c63837c5ea8d5e8f18fb0fb2a38abd4249dedca39ed0a1f
+MAINTAINER dvyukov@google.com
+
+RUN git clone --depth 1 https://github.com/curl/curl.git /src/curl
```

Comment:
You need to pin a version for curl as well.

Reply:
I left that to be done by `checkout_commit.py` because `oss-fuzz.yaml` has the commit. Should I blank that and put the version here instead ?

Reply:
Oh, ok that's fine.
Can you mention that in a comment here? might be confusing otherwise.

Reply:
Noted.